src/main/java/com/statisticalcalculator/StatisticalCalculatorController.java

```diff
@@ -3,6 +3,8 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+import com.statisticalcalculator.CalcDTO.SaveCalcDTO;

```

Comment:
Package names should be all lowercase (calcdto, not CalcDTO.  Even better, just call it dto, as calcdto is restrictive).